packages/forms/frontend/sirius-components-forms/src/form/FormEventFragments.ts

```diff
@@ -314,6 +314,14 @@ export const widgetFields = (contributions: Array<WidgetContribution>) => `
     stringValue: value
   }
 
+  fragment sliderFields on Slider {
+    label
+    iconURL
+    minValue
+    maxValue
+    currentValue
+  }
+
```

Comment:
Isn't the `... on Slider { ...sliderFields }` usage in `widgetFields` missing?

Reply:
Done